cluster-operator/src/test/java/io/strimzi/operator/cluster/model/KafkaBrokerConfigurationBuilderTest.java

```diff
@@ -79,7 +79,10 @@ public void testCruiseControl()  {
                 "cruise.control.metrics.reporter.ssl.keystore.password=${CERTS_STORE_PASSWORD}\n" +
                 "cruise.control.metrics.reporter.ssl.truststore.type=PKCS12\n" +
                 "cruise.control.metrics.reporter.ssl.truststore.location=/tmp/kafka/cluster.truststore.p12\n" +
-                "cruise.control.metrics.reporter.ssl.truststore.password=${CERTS_STORE_PASSWORD}"));
+                "cruise.control.metrics.reporter.ssl.truststore.password=${CERTS_STORE_PASSWORD}\n" +
+                "cruise.control.metrics.topic.auto.create=true\n" +
+                "cruise.control.metrics.topic.num.partitions=1\n" +
+                "cruise.control.metrics.topic.replication.factor=1"));
```

Comment:
Are we sure this is the right default setting?

Reply:
other than that I don't see it working. I tried but the AdminClient times out 3 times fetching metadata before creating the topic and it gives up. I will investigate more.

Reply:
Updated these to match the Kafka configuration default; CC doesn't specify a default for these parameters